src/constants.ts

```diff
@@ -1,4 +1,6 @@
 export const apiUrl = process.env.NEXT_PUBLIC_API_URI;
+export const pkbUrl = process.env.NEXT_PUBLIC_PKB_URI;
+export const pkbAuth = process.env.NEXT_PUBLIC_PKB_AUTH;
```

Comment:
So this is going to be in the frontend source code.
I know this is what also happens currently in JMP. But is there a more secure way to auth against pkb? @gabalese is setting up JWT auth with Auth0 in the saleor-api (which PKB is part of).

Reply:
I got this info from @gabalese. He told me that this endpoint is not using JWT (yet).

Reply:
As soon as it is available it should be a small  change to use auth0/JWT I suppose.